### PR TITLE
Add provided className to children of Wizard

### PR DESCRIPTION
**Pull request by osdiab**
**Jira:**
none, but this will help solve https://clever.atlassian.net/browse/CROC-902.

Discussion on Slack regarding why this is useful here: https://clever.slack.com/archives/C0ERS5ABB/p1497294991024954

**Overview:**
Use both the custom class name provided to the wizard as well as the `Wizard` classname for all classes internal to the wizard, so they can be styled without compound selectors

**Screenshots/GIFs:**
screen shot 2017-06-13 at 2 45 08 pm

**Testing:**
- [x] Unit tests
- Manual tests:
  - [x] Chrome
  - [ ] Safari
  - [ ] IE10

**Roll Out:**
- Before merging:
  - [ ] Updated docs
  - [ ] Bumped version in `package.json`
    - Breaking change? Run `npm version minor`
    - Backward compatible change? Run `npm version patch`
    - Only changing documentation? All good. Skip this step.
- After merging:
  - [ ] Deployed updated docs (`make deploy-docs`)

### CODE REVIEW

File: docs/WizardExample.jsx [hunk 1]
```diff
@@ -11,13 +11,20 @@ class AddressStep extends React.Component {
   render() {
     const {setWizardState, wizardState, recipient} = this.props;
     return (
-      <form ref={(f) => { this.form = f; }} onSubmit={(e) => e.preventDefault()}>
+      <form
+        ref={f => {
+          this.form = f;
+        }}
+        onSubmit={e => e.preventDefault()}
+      >
```
Comment (kofi-clever) [inline docs/WizardExample.jsx]: lot of noisy unrelated changes in here - is that intentional?

Comment (osdiab) [inline docs/WizardExample.jsx]: Ah must be the eslint/vscode auto fix on save setting I was using :P i'll try to go through and undo some of this stuff

Comment (osdiab) [inline docs/WizardExample.jsx]: Ah it was triggering since I set a max width of 80 chars on my own editor, I think it blindly applied it to everythign

Comment (osdiab) [inline docs/WizardExample.jsx]: Yep, totally was VSCode's auto format on save thing. Disabled that, repushed
